cli/src/commands/download.ts

```diff
@@ -78,8 +79,7 @@ export default class Download extends Command {
       this.conf.set('projects', projects);
     }
     this.log('Downloading application...');
-    const targetDir = '.';
-    const extract = tar.extract({ cwd: targetDir });
+    const extract = tar.extract({ cwd: '.' });
```

Comment:
Might be easier check before with `fs-extra` `emptyDir` to avoid the whole download just to fail at rename. (still a race of course).